src/Header/Header.css

```diff
@@ -0,0 +1,10 @@
+header {
+  text-align: center;
```

Comment:
I like our page! We might want to do a little refactoring of styling down the road.

Reply:
I agree. Refactoring and styling will be good.